### PR TITLE
blank space removed from meshery section

### LINKED ISSUE #3900: lots of blank space and no space between buttons in tablet view of meshery section.
### Current Behavior

lots of blank space and no space between buttons in tablet view of meshery section.

### Screenshots / Mockups

Screenshot 2023-03-06 at 12 27 06 AM

### Acceptance Tests

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by ShravaniAK**
**Description**

This PR fixes #3900 

**Notes for Reviewers**
earlier in tablet view 

Screenshot 2023-03-06 at 12 52 14 PM

now 

Screenshot 2023-03-06 at 12 51 31 PM

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e7744b9c2b99798252a2a3af716c992a072815f4 at: https://640599eddccd424ef0b127ff--layer5.netlify.app

File: src/sections/Meshery/meshery.style.js [hunk 1]
```diff
@@ -91,12 +91,19 @@ const MesheryWrapper = styled.section`
...
                 margin-top: 2rem;
 
+                
                 .meshery-hero-img {
+                    min-width:30rem;
                     text-align: center;
                     padding: 1rem 1rem 1rem 1rem;
                 }
-
+                .meshery-hero-img {
```
Comment (leecalcote) [inline src/sections/Meshery/meshery.style.js:100]: Am I reading this wrong or did you repeat the same class twice? If so, is there a special need to do so?

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Thank you for working on this, @ShravaniAK 👍

**Comment by ShravaniAK** [comment]
@leecalcote can you please review it once again , I have corrected the mistake and aligned text to center.

**Comment by l5io** [comment]
🚀 Preview for commit f78d05c8c613dce4c77e97a10944e42092d7d679 at: https://64094aa814f8464bd3d6b560--layer5.netlify.app

**Comment by swagataroy30** [comment]
Hey, some spacing is required between the two buttons and else looks good!

Screenshot from 2023-03-09 18-19-20

**Comment by swagataroy30** [comment]
The border radius of the meshery video is not consistent in smaller devices, just a suggestion i don't know if it's meant to be that way ... @leecalcote your views on it ?

Screenshot from 2023-03-09 18-37-51

**Comment by ShravaniAK** [comment]
@swagataroy30 Its not because of border radius , but width issue , i have corrected everything now , and thanks for informing .

**Comment by l5io** [comment]
🚀 Preview for commit 3e42a99d36dea1a2e18fd67ea0e5b9f053d6d233 at: https://6409e4deae02200a973d9cc9--layer5.netlify.app

File: src/sections/Meshery/meshery.style.js [hunk 2]
```diff
@@ -120,14 +133,42 @@ const MesheryWrapper = styled.section`
...
                 padding: 0;
                 .meshery-hero-img {
                     text-align: center;
                     padding: 0;
-                    min-width: 15rem;
+                    
+                }
+                .desc-text{
+                    text-align: center;
+                    margin-bottom: .75rem;
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/meshery.style.js:148]: You don't need to add same styles again for other media queries until and unless we are restricting the CSS upto a resolution, which we are not in this case. So, please o though the code and remove all duplicate declarations and verify everything looks good.

**Comment by l5io** [comment]
🚀 Preview for commit 5036f39041985b63837b6185cbe0b1de26bb4c8a at: https://640c762adcf74667d34b2c8c--layer5.netlify.app

**Comment by Chadha93** [comment]
@ShravaniAK  Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**